pkgs/development/libraries/science/math/amd-libflame/default.nix

```diff
@@ -56,7 +56,7 @@ stdenv.mkDerivation rec {
     description = "LAPACK-compatible linear algebra library optimized for AMD CPUs";
     homepage = "https://developer.amd.com/amd-aocl/blas-library/";
     license = licenses.bsd3;
-    maintainers = [ maintainers.markuskowa ];
```

Comment:
Please remove that commit. This is a non-change. Both forms a valid. No need to change it.

Reply:
fixed